modules/airgap/setup/windows_install.ps1

```diff
@@ -0,0 +1,48 @@
+param (
+    [string]$serverIP,
+    [string]$token,
+    [string]$nodeIP,
+    [string]$airgapMethod,
+    [string]$agentFlags
+)
+
+# Create dirs
+New-Item -Type Directory C:/etc/rancher/rke2 -Force
+New-Item -Type Directory C:/var/lib/rancher/rke2/agent/images/ -Force
+
+# Setting config
+Write-Host "Set config.yaml..."
+Set-Content -Path C:/etc/rancher/rke2/config.yaml -Value @"
+server: "https://$($serverIP):9345"
+token: "$($token)"
+node-ip: "$($nodeIP)"
+"@
+if (!$agentFlags) {
+    Add-Content -Path C:/etc/rancher/rke2/config.yaml -Value "$($agentFlags)"
+}
+
+# Setting env
+Write-Host "Set env path..."
+$env:PATH+=";c:\var\lib\rancher\rke2\bin;c:\usr\local\bin"
+[Environment]::SetEnvironmentVariable(
+    "Path",
+    [Environment]::GetEnvironmentVariable("Path", [EnvironmentVariableTarget]::Machine) + ";c:\var\lib\rancher\rke2\bin;c:\usr\local\bin;c:\var\lib\rancher\rke2\",
+    [EnvironmentVariableTarget]::Machine)
+
+# Checking airgap method
+if ($airgapMethod -like "private_registry") {
+    Write-Host "Copy registries.yaml..."
+    Copy-Item C:/Users/Administrator/registries-windows.yaml C:/etc/rancher/rke2/registries.yaml
+}
+if ($airgapMethod -like "tarball") {
+    Write-Host "Copy tarball images..."
+    Copy-Item C:/Users/Administrator/rke2-windows-ltsc2022-amd64-images.tar* C:/var/lib/rancher/rke2/agent/images/
+}
+
+# Copying assets and starting rke2 service
+Write-Host "Copy agent image..."
+Copy-Item C:/Users/Administrator/rke2.windows-amd64.tar.gz C:/var/lib/rancher/rke2/agent/images/
```

Comment:
We will need to use variable tarball_type instead as we have given an option to pass tarball_type here? https://github.com/rancher/distros-test-framework/pull/224/files#diff-ff375bab39080aa6e171f0af8a129de77bb3648abfa4fae351a5b8ed00286fdaR133

Reply:
The tarball type will be part of the tarball PR, this is only for private_registry. Also this image is only available as `windows-amd64.tar.gz`, it does not have the `tar.zst`